src/form-question-types.rst

```diff
@@ -1658,33 +1659,151 @@ In the parameters column, write ``max-pixels=`` followed by the desired maximum
+
+Recording status is also displayed below the audio question text. There is a time representing the current length of the recording (5) and a diagram (6) representing the volume of the recording over time. The diagram provides confirmation that the microphone is working and can help a user ensure an even, sufficient volume.
+
+Other questions can be included on the same screen as a built-in recording question. As shown in the screenshot above, this makes it possible to capture quantitative content while recording. To achieve this, put the questions in a :ref:`field list <field-list>`.
+
+During recording, the user is prevented from leaving the current question screen. However, it is safe to use other applications or to lock the device screen.
+
+Once recording is stopped, the control bar disappears. The recording is made available for playback below the question text.
+
+To replace the audio captured, first delete the current file and then record again.
+
+In some rare cases such as the device running out of space, the recording may complete successfully but not be attached to the form. If this happens, a dialog will be displayed explaining that the file is available but needs to be accessed manually. You can find these files in the ``recordings`` folder of the :ref:`Collect directory <collect-directory>`. This folder is never cleared so consider emptying it yourself once you have retrieved its files.
+
+.. _customizing-audio-quality:
+
+Customizing audio quality
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.29
+
+  `ODK Collect v1.29.0 <https://github.com/getodk/collect/releases/tag/v1.29.0>`_, `pyxform` v1.3.0, `XLSForm Online` v2.3.0
+
+The quality of audio recordings can be customized using the ``quality`` parameter. If a ``quality`` is specified, the built-in recorder is always used, regardless of Collect settings. If no ``quality`` is specified and Collect is :ref:`configured to use the built-in recorder <use-external-app-for-audio-recording>`, ``normal`` is used. The available quality values are:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Value
+     - Extension
+     - Encoding
+     - Bit rate
+     - Sample rate
+     - File size
+   * - normal
+     - .m4a
+     - AAC
+     - 64kbps
+     - 32kHz
+     - ~30MB/hour
+   * - low
+     - .m4a
+     - AAC
+     - 24kbps
+     - 32kHz
+     - ~11MB/hour
+   * - voice-only
+     - .amr
+     - AMR
+     - 12.2kbps
+     - 8kHz
+     - ~5MB/hour
+
+.. tip::
+
+  We'd recommend only using ``voice-only`` for one-on-one interviews in a quiet place as otherwise there might be too much detail loss. ``low`` will sound compressed but speech is generally intelligible, even if multiple people are talking at once. ``normal`` is similar to typical podcast settings and will sound good on most devices. If in doubt, it's a good idea to test the different qualities out to see which one fits your use case best.
```

Comment:
Not sure about "podcast quality" here. I'm guessing you're right that a lot of podcasts would be delivered at this kind of quality, but they'd probably be recorded with good mics/gear and then encoding. Maybe I'm overthinking?

Reply:
I see. Maybe another take is "``normal`` will sound good on most devices and the limiting quality factor will generally be the quality of the microphone and the recording environment rather than the encoding."

Reply:
Actually I like the use of "podcast" more and more as I think about it as it feels like it doesn't overpromise around non voice recordings. We're encouraging people to try them out which feels like the most important thing. Forget I said antyhing!

Reply:
Maybe there should be a tip or note saying that mic quality and recording environment are going to be limiting factors. I do think that's important to set expectations and also maybe give a hint to someone who's not getting the results they want that another device or an external mic could help.

Reply:
Ok to address that I've tried to add more emphasis to our note on testing things out. What do you think?